base/sparse/sparsematrix.jl

```diff
@@ -2458,11 +2461,14 @@ function setindex!{Tv,Ti,T<:Integer}(A::SparseMatrixCSC{Tv,Ti}, B::SparseMatrixC
     colptrB = B.colptr; rowvalB = B.rowval; nzvalB = B.nzval
```

Comment:
It seems you've removed all references to `colptrB`, `rowvalB`, and `nzvalB` in this function. Perhaps nix this line? Or did I miss remaining references?